### DISCUSSION TITLE
Change the default base url from static files.

### DISCUSSION THREAD

**Discussion by peculiarity** [vercel/next.js#24139] [Help]
Hello Commnity,

I won't get into details  why the configuration is like that but we have an application. There is CloudFront configuration that when requested `https://example.com/app` it loads the application ( Pay attention to the context path ). The problem arises because when the application loads, all static files are loaded from `https://example.com/static/..`.  According to the documentation for [Static Files Serving](https://nextjs.org/docs/basic-features/static-file-serving) all the files are served from `/`.

I would like to change that path so that all of my staticf files are served from `/app/static/...`.  How do I do that?

Edit: Is there a way to actually configure this or do I have to manually point out?

Thank you all in advance!

**Comment by nalbertcerqueira** [comment]
Hii good morning !

You can use the base path config in next.config.js. Check the link above to see more about it .

https://nextjs.org/docs/api-reference/next.config.js/basepath
